### PR TITLE
feat(quickstart): Add examples for all quickstart platforms

### PR DESCRIPTION
**Pull request by EvanPurkhiser**
Just for the quickstart page, adds a 'sending a verification event' section.

### CODE REVIEW

**Comment by eurica** [comment]
@EvanPurkhiser This is the change that tells new projects how to send an easy error?  

I'd love to do more later, but this is an improvement.  I say :shipit:

**Comment by EvanPurkhiser** [comment]
There are a few more, but yeah this does some. Not all platform docs work the same way. I have quite a few PRs right now that do changes, but you can see all the ones that DON'T tell you here https://github.com/getsentry/sentry-docs/pull/1034

File: src/collections/_documentation/error-reporting/getting-started-verify/aspnetcore.md [hunk 1]
```diff
@@ -0,0 +1 @@
+See the [provided examples in the `dotnet` SDK repository](https://github.com/getsentry/sentry-dotnet/tree/master/samples) for simple examples to send your first event to Sentry.
```
Comment (MimiDumpling) [inline src/collections/_documentation/error-reporting/getting-started-verify/aspnetcore.md]:
```suggestion
See the [provided examples in the `dotnet` SDK repository](https://github.com/getsentry/sentry-dotnet/tree/master/samples) for examples to send your first event to Sentry.
```

File: src/collections/_documentation/error-reporting/getting-started-verify/browser.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+A simple way to break your JavaScript application is to call an undefined function:
```
Comment (MimiDumpling) [inline src/collections/_documentation/error-reporting/getting-started-verify/browser.md]:
```suggestion
One way to break your JavaScript application is to call an undefined function:
```

File: src/collections/_documentation/error-reporting/getting-started-verify/cordova.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+A simple way to break your Cordova application is to call an undefined function:
```
Comment (MimiDumpling) [inline src/collections/_documentation/error-reporting/getting-started-verify/cordova.md]:
```suggestion
One way to break your Cordova application is to call an undefined function:
```

File: src/collections/_documentation/error-reporting/getting-started-verify/csharp.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+You can easily verify Sentry is capturing unhandled exceptions by raising an exception, for example you can use the following snippet to raise a `DivideByZeroException`:
```
Comment (MimiDumpling) [inline src/collections/_documentation/error-reporting/getting-started-verify/csharp.md]:
```suggestion
You can verify Sentry is capturing unhandled exceptions by raising an exception. For example, you can use the following snippet to raise a `DivideByZeroException`:
```

File: src/collections/_documentation/error-reporting/getting-started-verify/electron.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+A simple way to break your Electron application is to call an undefined function:
```
Comment (MimiDumpling) [inline src/collections/_documentation/error-reporting/getting-started-verify/electron.md]:
```suggestion
One way to break your Electron application is to call an undefined function:
```

File: src/collections/_documentation/error-reporting/getting-started-verify/node.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+A simple way to break your JavaScript application is to call an undefined function:
```
Comment (MimiDumpling) [inline src/collections/_documentation/error-reporting/getting-started-verify/node.md]:
```suggestion
One way to break your JavaScript application is to call an undefined function:
```

File: src/collections/_documentation/error-reporting/getting-started-verify/php.md [hunk 7]
```diff
@@ -0,0 +1,5 @@
+You can easily trigger a PHP exception by throwing one in your application:
```
Comment (MimiDumpling) [inline src/collections/_documentation/error-reporting/getting-started-verify/php.md]:
```suggestion
You can trigger a PHP exception by throwing one in your application:
```

File: src/collections/_documentation/error-reporting/getting-started-verify/python.md [hunk 8]
```diff
@@ -0,0 +1,6 @@
+You can easily cause a python error by inserting a divide by zero expression
```
Comment (MimiDumpling) [inline src/collections/_documentation/error-reporting/getting-started-verify/python.md]:
```suggestion
You can cause a Python error by inserting a divide by zero expression
```

File: src/collections/_documentation/error-reporting/getting-started-verify/rust.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
+The quickets way to verify Sentry in your rust application is to cause a panic:
```
Comment (MimiDumpling) [inline src/collections/_documentation/error-reporting/getting-started-verify/rust.md]:
```suggestion
The quickest way to verify Sentry in your Rust application is to cause a panic:
```

File: src/collections/_documentation/error-reporting/quickstart.md [hunk 10]
```diff
@@ -56,6 +56,14 @@ Note: If you’re using Heroku, and you’ve added Hosted Sentry via the standar
...
+
+Once you have Sentry integrated into your project, you probably want to verify that everything is working as expected before deploying it, and what better way to do that than to break your application!
+
+{% wizard %}
+{% include components/platform_content.html content_dir='getting-started-verify' %}
+{% endwizard %}
+
 ## Next Steps
 
 Now that you’ve got basic reporting setup, you’ll want to explore adding additional context to your data.
```
Comment (MimiDumpling) [inline src/collections/_documentation/error-reporting/quickstart.md]:
```suggestion
Now that you’ve got basic reporting set up, you’ll want to explore adding additional context to your data.
```

**Pull request merged**